config/locales/en.yml

```diff
@@ -506,6 +509,52 @@ en:
       save_settings: "Save settings"
       minimum_listing_price_below_tx_fee: "The minimum transaction size has to be greater than or equal to the minimum transaction fee: %{minimum_transaction_fee}."
       minimum_listing_price_below_min: "The minimum transaction size has to be greater than the minimum commission %{minimum_commission}."
+    payment_preferences:
+      title: Payment system preferences
+      integration_info_text_paypal_and_stripe: "<p>If you want to allow your users to accept payments, you  need to connect at least one of the two payment providers supported by Sharetribe: Stripe and PayPal. You can choose to use both, which means yout users will be able to accept payments with either payment method.</p> You ca create a new account with either service if you don't have one already. Once you have connected an account, you can choose a minimum transaction size and a possible commission fee.</p> What is a difference between Stripe and PayPal? Stripe allows you to delay payout to the seller up to 3 months and offeres one payment method (credit card). Paypal provide two payment methods (credit card and PayPal account) and offers a protection program for buyers and sellers</p>"
```

Comment:
There shouldn't be any html-tags in translations. These end up in webtranslateit and all translators are then required to replicate them across the translations. This means that a single translator that forgets to put in a closing tag or puts in an incomplete tag like `</p` breaks the whole layout for that language.

For that, and security reasons, the translations should be handled like user inputs and sanitized accordingly. If tags are necessary, then those need to be built in backend.